core/forms.py

```diff
@@ -56,10 +56,15 @@ class InvitationForm(forms.ModelForm):
         help_text='This message will be shown to the user you invite.',
         widget=forms.Textarea,
     )
+    is_open = forms.BooleanField(
```

Comment:
invitation form should feature open invite attribute. Not sure if this is the best way to do that